README.md

```diff
@@ -52,6 +52,11 @@ For example, you can add the following line to your shell’s rc file (e.g. ~/.b
 ```export PATH=<mongodb-install-directory>/bin:$PATH```
 Replace <mongodb-install-directory> with the path to the extracted MongoDB archive.
 
+## On Linux
+For Ubuntu,SUSE, and Debian systems, you can install mongoDB Community Edition using .deb packages. The process varies from distro to distro, so I suggest you visit the link below, select your distribution.
```

Comment:
You could have added commands to install MongoDB on those mentioned distros.

Reply:
On Ubuntu MongoDB can be installed simply by , 
`sudo apt install mongodb`

Reply:
Ok, I'll add those commands for those distros

Reply:
Done! Thanks for pointing that out. I used the terminal command `sudo apt-get install -y mongodb-org` so they get the latest stable version

Reply:
Done! I used `sudo apt-get install -y mongodb-org` so they get the latest stable version :)